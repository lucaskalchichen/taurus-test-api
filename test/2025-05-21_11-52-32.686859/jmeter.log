2025-05-21 11:52:36,251 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 11:52:36,261 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 11:52:36,261 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 11:52:36,262 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859/jmeter-bzt.properties
2025-05-21 11:52:36,262 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859/system.properties
2025-05-21 11:52:36,265 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 11:52:36,266 INFO o.a.j.JMeter: Version 5.5
2025-05-21 11:52:36,266 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 11:52:36,266 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 11:52:36,266 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 11:52:36,266 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 11:52:36,266 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 11:52:36,267 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 11:52:36,267 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 11:52:36,267 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 11:52:36,267 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 11:52:36,269 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 11:52:36,270 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 11:52:36,270 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 11:52:36,270 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:52:36,270 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:52:36,276 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 11:52:36,276 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859
2025-05-21 11:52:36,276 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859
2025-05-21 11:52:36,277 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859
2025-05-21 11:52:36,283 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 11:52:36,283 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859'
2025-05-21 11:52:36,358 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 11:52:36,368 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 11:52:36,370 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 11:52:36,371 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859/modified_requests.jmx
2025-05-21 11:52:36,387 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 11:52:36,388 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 11:52:36,394 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:52:36,394 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:52:36,395 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:52:36,395 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:52:36,395 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 11:52:36,395 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 11:52:36,411 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 11:52:36,412 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:52:36,412 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:52:36,418 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 11:52:36,418 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 11:52:36,551 INFO o.a.j.JMeter: Running test (1747839156551)
2025-05-21 11:52:36,560 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 11:52:36,560 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 11:52:36,560 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 11:52:36,560 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 11:52:36,566 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 11:52:36,578 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 11:52:36,579 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 11:52:36,579 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 11:52:36,581 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 11:52:36,581 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 11:52:36,581 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 11:52:36,582 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 11:52:36,582 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 11:52:36,634 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 11:52:36,703 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 11:52:36,776 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 11:52:36,848 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 11:52:36,917 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 11:52:36,988 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 11:52:37,058 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 11:52:37,129 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 11:52:37,200 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 11:52:37,271 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 11:52:37,342 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 11:52:37,414 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 11:52:37,485 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 11:52:37,555 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 11:52:37,627 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 11:52:37,697 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 11:52:37,768 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 11:52:37,838 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 11:52:37,910 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 11:52:37,981 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 11:52:38,052 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 11:52:38,122 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 11:52:38,194 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 11:52:38,266 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 11:52:38,335 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 11:52:38,407 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 11:52:38,478 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 11:52:38,550 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 11:52:38,620 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 11:52:38,691 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 11:52:38,738 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 11:52:38,739 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 11:52:38,739 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 11:52:38,739 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 11:52:38,740 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 11:52:38,740 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 11:52:38,743 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 11:52:38,743 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 11:52:38,747 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 11:52:38,748 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 11:52:38,750 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 11:52:38,750 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 11:52:38,751 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 11:52:38,751 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 11:52:38,751 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 11:52:38,751 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 11:52:38,752 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 11:52:38,752 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 11:52:38,752 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 11:52:38,752 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 11:52:38,753 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 11:52:38,753 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 11:52:38,753 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 11:52:38,753 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 11:52:38,754 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 11:52:38,754 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 11:52:38,754 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 11:52:38,754 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 11:52:38,755 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 11:52:38,755 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 11:52:38,755 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 11:52:38,755 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 11:52:38,756 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 11:52:38,756 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 11:52:38,756 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 11:52:38,756 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 11:52:38,757 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 11:52:38,757 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 11:52:38,757 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 11:52:38,757 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 11:52:38,758 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 11:52:38,758 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 11:52:38,758 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 11:52:38,758 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 11:52:38,759 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 11:52:38,759 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 11:52:38,759 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 11:52:38,759 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 11:52:38,760 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 11:52:38,760 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 11:52:38,760 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 11:52:38,760 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 11:52:38,761 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 11:52:38,761 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 11:52:38,761 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 11:52:38,761 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 11:52:38,762 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 11:52:38,762 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 11:52:38,763 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 11:52:38,763 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 11:52:38,763 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 11:52:38,763 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 11:52:38,763 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 11:52:38,766 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 11:52:38,766 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 11:52:38,832 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 11:52:38,837 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 11:52:38,837 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 11:52:38,903 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 11:52:38,908 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 11:52:38,909 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 11:52:38,974 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 11:52:38,981 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 11:52:38,981 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 11:52:39,046 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 11:52:39,054 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 11:52:39,054 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 11:52:39,116 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 11:52:39,122 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 11:52:39,122 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 11:52:39,188 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 11:52:39,193 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 11:52:39,194 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 11:52:39,258 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 11:52:39,264 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 11:52:39,264 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 11:52:39,332 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 11:52:39,336 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 11:52:39,336 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 11:52:39,401 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 11:52:39,409 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 11:52:39,409 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 11:52:39,472 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 11:52:39,481 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 11:52:39,481 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 11:52:39,481 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 11:52:39,483 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
