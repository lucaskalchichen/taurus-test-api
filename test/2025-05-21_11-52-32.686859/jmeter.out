Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-52-32.686859/modified_requests.jmx
Starting standalone test @ 2025 May 21 11:52:36 ART (1747839156407)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 11:52:39 ART (1747839159482)
... end of run
